client/signup/steps/plans/tabbed-plans.js

```diff
@@ -0,0 +1,986 @@
+				description: translate( 'Professional email marketing integration.' ),
+			},
+		],
+	};
+
+	const [ featureComparison, setFeatureComparison ] = useState(
+		featureComparisonData.Professional
+	);
+	const [ planDetails, setPlanDetails ] = useState();
+	const [ selectedTab, setSelectedTab ] = useState( 'Professional' );
+	const [ termLength, setTermLength ] = useState( 'yearly' );
+	const [ showFeatures, setShowFeatures ] = useState( false );
+
+	const toggleTab = () =>
+		selectedTab === tabList[ 0 ] ? setSelectedTab( tabList[ 1 ] ) : setSelectedTab( tabList[ 0 ] );
+
+	const toggleTerm = () =>
+		termLength === 'yearly' ? setTermLength( 'monthly' ) : setTermLength( 'yearly' );
+
+	const handleUpgradeButtonClick = ( productSlug, productId ) => {
+		const args =
+			productSlug === 'free_plan' ? null : { product_slug: productSlug, product_id: productId };
+		onUpgradeClick( args );
+	};
+
+	const tabHandler = ( event ) => {
+		if ( event.code === 'Tab' ) {
+			event.preventDefault();
+			toggleTab();
+		}
+	};
+
+	useEffect( () => {
+		const planFilter =
+			selectedTab === 'Professional'
+				? [ 'Premium', 'Business', 'eCommerce' ]
+				: [ 'Free', 'Personal' ];
+
+		const displayedPlans = planProperties
+			.filter( ( plan ) => planFilter.includes( plan.planName ) )
+			.filter( ( plan ) => plan.termLength === termLength || plan.termLength === null );
+
+		// Remove free plan if the user has selected a domain
+
+		setFeatureComparison( featureComparisonData[ selectedTab ] );
+		setPlanDetails( displayedPlans );
+	}, [ selectedTab, planProperties, termLength ] );
+
+	useEffect( () => {
+		window.addEventListener( 'keydown', tabHandler );
+
+		return () => {
+			window.removeEventListener( 'keydown', tabHandler );
+		};
+	}, [ selectedTab ] );
+
+	return (
+		<>
+			<TabWrapper>
+				<Tabs>
+					{ tabList.map( ( item, index ) =>
+						item === selectedTab ? (
+							<SelectedTab
+								key={ `tab${ index }` }
+								className={ `tabbed-plans__tab-${ index + 1 }` }
+								onClick={ toggleTab }
+							>
+								{ item }
+							</SelectedTab>
+						) : (
+							<Tab
+								key={ `tab${ index }` }
+								className={ `tabbed-plans__tab-${ index + 1 }` }
+								onClick={ toggleTab }
+							>
+								{ item }
+							</Tab>
+						)
+					) }
+				</Tabs>
+			</TabWrapper>
+			<Grid className="tabbed-plans__grid-container">
+				<TermToggles className="tabbed-plans__term-toggles">
+					<RadioButton>
+						<input
+							type="radio"
+							checked={ termLength === 'yearly' ? 'checked' : '' }
+							name="monthly"
+							onClick={ toggleTerm }
+						/>
+						<Checkmark></Checkmark>
+						<span>{ translate( 'Pay Yearly' ) }</span>
+					</RadioButton>
+					<RadioButton>
+						<input
+							type="radio"
+							checked={ termLength === 'monthly' ? 'checked' : '' }
+							name="monthly"
+							onClick={ toggleTerm }
+						/>
+						<Checkmark></Checkmark>
+						<span>{ translate( 'Pay Monthly' ) }</span>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **28** times:
`translate( 'Pay monthly' )` *ES Score: 12*